### PR TITLE
Added Signup Form

### PR DESCRIPTION
**Pull request by cocojoe**

### CODE REVIEW

File: 01-Custom-Form/app/components/SignupForm.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        super(props);
+        this.state = { username: "", password: "", usernameError: false, passwordError: false }
+    }
+
+    validateSignup = () => {
+
+        usernameError = false
+        passwordError = false
+
+        if (!this.state.username.length) {
```
[the remark this answers is not in the document]
Comment (cocojoe) [reply 01-Custom-Form/app/components/SignupForm.js]: Seems fine

**Comment by cocojoe** [comment]
simulator screen shot - iphone 6 - 2017-10-27 at 14 40 00

File: 01-Custom-Form/app/components/SignupForm.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (luisrudge) [reply 01-Custom-Form/app/components/SignupForm.js]: !0 === false

[the remark this answers is not in the document]
Comment (luisrudge) [reply 01-Custom-Form/app/components/SignupForm.js]: gotta love js 😝

**Pull request merged**